backend/package.json

```diff
@@ -6,6 +6,7 @@
   "scripts": {
     "build": "tsc",
     "start": "npm run migration:migrate && node --enable-source-maps dist/index.js",
+    "start-local": "npm run migration:migrate && rm -rf ./dist && tsc && node --enable-source-maps dist/index.js",
```

Comment:
We have the `start:dev` command, was there another motivation behind this?

Reply:
did not see `start:dev`. removed `start-local`.